components/transaction-operation/index.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import styles from './transaction-operation.module.css';
+
+const TransactionOperation = (props) => {
+  const { modal, showModal, personData, receivers, transactionType } = props;
+
+  return (
+    <>
+      {' '}
+      {modal ? (
+        <div className={styles.background}>
```

Comment:
Why not also add a backdrop when the modal opens and we also close the modal upon clicking on the backdrop